src/plugins/vis_types/vislib/public/vislib/vis.js

```diff
@@ -28,7 +28,7 @@ export class Vis extends EventEmitter {
     super();
     this.element = element.get ? element.get(0) : element;
     this.visConfigArgs = _.cloneDeep(visConfigArgs);
-    this.visConfigArgs.dimmingOpacity = core.uiSettings.get(DIMMING_OPACITY_SETTING);
+    this.visConfigArgs.dimmingOpacity = 0.5;
```

Comment:
Looks like `dimmingOpacity` is never read, would it be save to remove it?

Reply:
It is, you are right!